sdk/samples/iot/hub/linux/how_to_iot_hub_samples_linux.md

```diff
@@ -150,31 +153,29 @@ _The following was run on an Ubuntu Desktop 18.04 environment, but it also works
     Save the certificate Fingerprint above (in this example, `C8DC8780C64FFBB5A66D8BC5D39D3C1BBB03FB69`).
     It will be used to create the logical device on your Azure IoT Hub.
 
-
-06. Create a logical device
+6. Create a logical device
 
     - Log into your Azure account on [Azure Portal](https://portal.azure.com)
     - On the menu on the left, click on "IoT devices" under "Explorers".
     - Click on "New".
     - Type an unique ID for your device.
-    - Select "X.509 Self-Signed" for "Authentication type". 
+    - Select "X.509 Self-Signed" for "Authentication type".
     - Type the fingerprint obtained in the previous step (the same can be used for primary and secondary Thumbprints; no colons!).
     - Click on "Save".
 
-07. Collect information about Azure IoT Hub and device
+7. Collect information about Azure IoT Hub and device
 
     For the Azure IoT Embedded SDK for C samples, we will need the Azure IoT Hub name and device ID.
 
     - Get the Azure IoT Hub FQDN.
```

Comment:
Is `FQDN` a commonly known term for folks who work in this domain? It isn't defined anywhere. Ideally, it would be written out fully  at least once in the docs (`fully qualified domain name`), maybe here:

https://github.com/Azure/azure-sdk-for-c/blob/3cb74d2067fce41a65310044d4a8f629e382e158/sdk/docs/iot/README.md#iot-hub-client-with-mqtt-stack
```C
static az_span my_iothub_hostname = AZ_SPAN_LITERAL_FROM_STR("<your hub fqdn here>");
```

Reply:
yes it's pretty commonly used. TBH if someone encounters that, they probably wouldn't look in the docs for the definition. A quick search online would pull up quicker and be correct.